### DISCUSSION TITLE
teoria relazionale - 4

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#71] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/f3d41aea-84ee-4e01-bc3e-88936b0acc8d)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/e6596770-f6a5-4a89-91f7-31c69ba07e86)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/dddaa98e-e40f-44f3-8e77-545c25e1a573)

**Reply by 7Andrei** [reply]
Per la decomposizione p non devi anche aggiungere una chiave per far si che ha una join senza perdite?

**Reply by Frisbon** [reply]
Hai ragione, grazie.
